src/reference.js

```diff
@@ -3,32 +3,36 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import { registerWidget } from '@nextcloud/vue/dist/Components/NcRichText.js'
+import { registerWidget } from '@nextcloud/vue/components/NcRichText'
 
 registerWidget('integration_github_issue_pr', async (el, { richObjectType, richObject, accessible }) => {
```

Comment:
wdyt of hiding the interactive button here and in the other registerWidget function below?